apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo/gradle/internal/DefaultCompilationUnit.kt

```diff
@@ -89,10 +89,7 @@ abstract class DefaultCompilationUnit @Inject constructor(
     }
   }
 
-  fun generateKotlinModels(): Boolean = when {
-    project.isKotlinMultiplatform -> true
-    else -> generateKotlinModels.orElse(service.generateKotlinModels).orElse(apolloExtension.generateKotlinModels).getOrElse(false)
-  }
+  fun generateKotlinModels(): Boolean = true
```

Comment:
Can we not inline this function too? Do we still need it?

Reply:
I kept the Gradle plugin java codegen path intact for now since it can ultimately be reused. I'd really like to remove this function though and not having to build the java/kotlin task graphs based on user input (i.e. certainly have different plugin ids for java and kotlin). Not 100% sure yet how this is going to end so this is a very rough first step.